test/testCLI.ts

```diff
@@ -78,4 +78,34 @@ export function run() {
     t.snapshot(readFileSync('./ReferencedType.d.ts', 'utf-8'))
     unlinkSync('./ReferencedType.d.ts')
   })
+
+  test('files in (-i), files out (-o)', t => {
```

Comment:
Should we add a test case that we concat all generated TS into one file when the input is a directory, while the output is a filename? eg. `-i a/**/*.json -o b.json` should concatenate all generated TS into `b.json`.

Reply:
I considered this feature case out of scope for this PR. If you think otherwise I can try to implement it

Reply:
It seems fair to me to leave it out of scope. If you choose to do that, would you mind at least detecting that case and throwing a `ReferenceError` when we hit it?

Reply:
I'll also open a new issue to track that feature request so others could contribute it 😄

Reply:
Simple concatenation doesn't work. For example if you have 3 schemas.

> A, B, C
> A refers definition in C
> B refers definition in C

In the output you'll have 3 files defining the same definition from C. If you just concatenate files, you'll get duplicate declaration errors.

Many to one file support would be very handy.

Reply:
That’s a really good point — though TS will merge interfaces for us, it will error when multiple files emit type aliases with the same name.

To fix that, we’ll need a way to de-dupe across files, ideally.